lsp/matlab_ls.lua

```diff
@@ -0,0 +1,25 @@
+---@brief
+---
+--- https://github.com/mathworks/MATLAB-language-server
+---
+--- MATLAB® language server implements the Microsoft® Language Server Protocol for the MATLAB language.
+return {
+  cmd = { 'matlab-language-server', '--stdio' },
+  filetypes = { 'matlab' },
+  root_dir = function(_, callback)
+    local root_dir = vim.fs.root(0, '.git')
```

Comment:
```suggestion
  root_dir = function(bufnr, callback)
    local root_dir = vim.fs.root(bufnr, '.git')
```

I'm not 100% sure about this. But I suspect things like `nvim foo.m bar.m` (open multiple files) will be incorrect since it will attempt to attach to `foo.m` twice (since `0` refers to the currently active buffer, not the buffer being attached).


Regardless, I think it's a good practice.